docs/index.md

```diff
@@ -189,13 +239,21 @@ In this example, assume a company has three Astra DB organizations:
 * A wholesale org
 * An internal usage org
 
-Follow these steps:
+*IMPORTANT*: As noted previously in this topic, here are the differences between `standard` and `sidecar` caller modes:
```

Comment:
Is there a reason this isn't written as a normal admonition?

Reply:
It's markdown...